spec/editors_draft.html

```diff
@@ -3298,6 +3298,261 @@ <h2>Evaluation Semantics</h2>
+      </p>
+
+      <ul>
+        <li>The parsing of a SPARQL Query string [[SPARQL11-QUERY]] or of a SPARQL Update string [[SPARQL11-UPDATE]], which may contain literals.</li>
+        <li>The loading of RDF data via a SPARQL <a data-cite="SPARQL11-UPDATE#load">`LOAD`</a> operation [[SPARQL11-UPDATE]]</li>
+        <li>The loading of RDF data via a single SPARQL graph store protocol request [[SPARQL11-HTTP-RDF-UPDATE]]</li>
+        <li>The loading of RDF data via any single SPARQL <a data-cite="SPARQL11-QUERY#unnamedGraph">`FROM`</a> or <a data-cite="SPARQL11-QUERY#namedGraphs">`FROM NAMED`</a> clause [[SPARQL11-QUERY]] in systems that <a data-cite="SPARQL11-SERVICE-DESCRIPTION#sd-dereferencesuris">support the dereferencing and loading of RDF data specified for the dataset to be queried</a></li>
+        <li>Importing of any RDF data to the internally-managed RDF dataset via means other than SPARQL Update statements [[SPARQL11-UPDATE]] or the SPARQL Graph Store HTTP Protocol [[SPARQL11-HTTP-RDF-UPDATE]]</li>
+        <li>The loading of a SPARQL result obtained from a SPARQL service when evaluating a <a data-cite="SPARQL11-FEDERATED-QUERY#service">`SERVICE`</a> pattern [[SPARQL11-FEDERATED-QUERY]]</li>
+      </ul>
+
+      <p>
+        As an example, consider the Turtle document in <a href="#load-renaming-blank-example-simple"></a>.
+        Loading this document MUST result in a system-internal representation of a single triple whose object is a <a>cdt:List literal</a> containing a blank node that, in the lexical form of the literal, is captured by a blank node identifier that is <em>not</em> "b0" (e.g., <a href="#load-renaming-blank-example-simple-result"></a>).
+      </p>
+      <pre id="load-renaming-blank-example-simple"
+         title="Loading of a cdt:List literal with a blank node"
+         data-transform="updateExample"
+         data-content-type="text/turtle"
+         class="nohighlight example">
+      <!--
+      ex:s ex:p1 "[ _:b0 ]"^^cdt:List .
+      -->
+      </pre>
+      <pre id="load-renaming-blank-example-simple-result"
+         title="Loading of a cdt:List literal with a blank node: Result"
+         data-transform="updateExample"
+         data-content-type="text/n-triples"
+         class="nohighlight example">
+      <!--
+      <http://example.org/s> <http://example.org/p1> "[ _:b001 ]"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/List> .
+      -->
+      </pre>
+
+      <p>
+        Evaluating a SPARQL Update in <a href="#load-renaming-blank-example-load-twice"></a> with two `INSERT DATA` operations with the same blank-node-containing <a>cdt:List literal</a> MUST result in a system-internal representation of two triples whose objects are <a>cdt:List literals</a> containing different blank nodes that, in the lexical form of these literals, are captured by different blank node identifiers (e.g., <a href="#load-renaming-blank-example-load-twice-result"></a>).
```

Comment:
"a SPARQL Update" ? Something seems to be missing here. Did you want to say "a SPARQL Update request" or "a SPARQL Update string" or something else?

Another issue: if these two `INSERT DATA` operations are indeed within the same request string, shouldn't the two occurrences of blank node identifier `_:b0` be considered to refer to the same blank node? The bullet points above are not entirely unambiguous about it (as they just mention "a SPARQL Update string").

Reply:
Interesting. I had figured the natural interpretation would be that the two inserts would result in differing internal blank nodes. There's no existing equivalent since two `INSERT DATA` patterns [cannot share blank node identifiers](https://www.w3.org/TR/sparql11-query/#grammarBNodes). Given the current restriction, do you have a preference for aligning with it, or in explicitly allowing co-reference from CDTs in different INSERT DATA patterns?

Reply:
Will update "Evaluating a SPARQL Update in" to "Evaluating the SPARQL Update request in".

Reply:
I see. The SPARQL spec says: "The same blank node label can not be used in [...] two INSERT DATA operations within a single SPARQL Update request." So, if we assume that this constraint extends to blank node identifiers in CDT literals, then the example with the two INSERT DATA operations is *syntactically* incorrect. Hence, it is not so suitable for the purpose of providing examples that demonstrate our import requirements. A better example would have two separate SPARQL Update requests with the same INSERT DATA operation.

Reply:
Another edit suggestion based on your response for this part.
```suggestion
        Evaluating the SPARQL Update request in <a href="#load-renaming-blank-example-load-twice"></a> with two `INSERT DATA` operations with the same blank-node-containing <a>cdt:List literal</a> MUST result in a system-internal representation of two triples whose objects are <a>cdt:List literals</a> containing different blank nodes that, in the lexical form of these literals, are captured by different blank node identifiers (e.g., <a href="#load-renaming-blank-example-load-twice-result"></a>).
```

Reply:
Regarding the example with the two INSERT DATA statements, I have fixed the example based on [my earlier comment](https://github.com/awslabs/SPARQL-CDTs/pull/2#discussion_r1622802293), and I have added a Note talking about the invalid case that we had in the previous version of this example. See commit https://github.com/awslabs/SPARQL-CDTs/pull/2/commits/d9f037dd50ee49d9ca14ea645e0ff3bf29685846